doc/whatsnew/fragments/9255.breaking

```diff
@@ -0,0 +1,9 @@
+Commented out code blocks will no longer emit fixme's, such as the following:
+```
+# typical comment # TODO: will not emit!
+# if foo:
+#    bar() # TODO: will not emit!
+#    baz()
+```
```

Comment:
```suggestion
Commented out code blocks such as ``#    bar() # TODO: remove dead code`` will no longer emit ``fixme``.
```
The regex is unforgiving, I suggest we simplify the release note. Beside the following would not emit, right ?
```
# typical comment # TODO: will not emit!
if foo:
   #    bar() # TODO: will not emit?
   baz()
```

Reply:
Yes, that's right. Thanks for the suggestion. I think it's a lot more succinct 👍